### ISSUE TITLE
Checks 72a & 72b

### ISSUE THREAD

**Issue by Richard-Fowle** [iho-ohi/ENCWG-S58-ENC-Validation-Checks#46]
From Tom De Puyt (ESRI)

![image](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/assets/48444903/4aa36387-8364-4ca5-b7e4-1908e11079ce)

I think, 72b makes 72a obsolete (for structure/equipment), because, a hierarchy of more than 2 levels does not exist without an equipment feature also being a structure feature for the next level ?  Consider A = structure, B = equipment. In order for the loop to complete, B has to be structure for C thus violating check 72b.   72a may still apply for collection features but I think that needs to be explicitly defined.
![image](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/assets/48444903/e442fa1a-9f1e-4758-a112-8d05c4fd18f9)

**Comment by Richard-Fowle** [comment]
I agree with Tom's observation that 72b covers stated example (for example master object is slave of its own slave) and that check 72a could still apply to collection objects (C_ASSO & C_AGGR) so I suggest amending the wording of check 72a :-

![image](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/assets/48444903/e69f1d24-bf6d-4573-8171-aaa7cdb974f1)

**Comment by FrankHippmann** [comment]
I can see the non-sense example of:

BCNLAT is master of C_ASSO, which references the BCNLAT as a peer.

Of course C_ASSO/C_AGGR should not be allowed to be part of a master slave relationship, but I can't find a specific check limiting the feature classes that can have the master role and slave roles. Perhaps we need a check stating that features having the master role need to be structures, and features having the slave role need to be equipment.

**Comment by Richard-Fowle** [comment]
Following discussion at ENCWG10 it was agreed to close this issue as it was considered not to be a problem that requires change.
